azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/ManagementClientAsync.java

```diff
@@ -912,6 +918,7 @@ public ManagementClientAsync(URI namespaceEndpointURI, ClientSettings clientSett
     /**
      * Deletes the queue described by the path relative to the service namespace base address.
      * @param path - The name of the entity relative to the service namespace base address.
+     * @return A completable future.
```

Comment:
>A completable future. [](start = 15, length = 21)

What completablefuture? More description please. A completable future that completes when the queue is deleted or something. Most developers read javadoc comments, Intellisense of editors also show javadoc to the developers. People really appreciate good javadocs. I heard most of Java platforms API documentation is written by developers, without which we can't write good code.